frontend/src/components/PresetList.tsx

```diff
@@ -0,0 +1,76 @@
+"use client"
+
+import React, { ReactNode } from "react"
```

Comment:
Ditto, don't import all of React, and don't annotate component functions with a return type, there's no need

Reply:
I always have some preference to precise the return type for readability, but okey removed